source/numerics/vector_tools_project.inst.in

```diff
@@ -67,3 +67,40 @@ for (VEC : SERIAL_VECTORS ; deal_II_dimension : DIMENSIONS; deal_II_space_dimens
     \}
 #endif
 }
+
+for (deal_II_dimension : DIMENSIONS)
```

Comment:
> We could in principle add a new list to template-arguments.h.in and use that for instantiating

i would really prefer not to do it... I am aware of definitions in `cmake/config/template-arguments.in`, sorry for not mentioning it at the beginning.

@tamiko :  you are probably the only one who knows the answer ;-)